### PR TITLE
add CultureInfo for Turkish OS

### LINKED ISSUE #4879: LogLevel error on Turkish Window OS.
Hi,

I'm getting an error when running Akka.dll v1.4.18 on Turkish Windows 10 Ent. OS. One of ToUpper function transforms lowercase "warning" log level to "WARNİNG". I tried to use ToUpper with English CultureInfo param and it worked without a problem.

`logLevel = logLevel.ToUpper(new System.Globalization.CultureInfo("en-EN"));`

https://github.com/akkadotnet/akka.net/blob/21762b90ac1d7db670487dda1368d84c61d693e5/src/core/Akka/Event/Logging.cs#L263-L291

```
[Fatal][05:41:37.037] System.ArgumentException
Unknown LogLevel: "WARNİNG". Valid values are: "DEBUG", "INFO", "WARNING", "ERROR" (Parameter 'logLevel')
   at Akka.Event.Logging.LogLevelFor(String logLevel)
   at Akka.Event.LoggingBus.StartDefaultLoggers(ActorSystemImpl system)
   at Akka.Actor.LocalActorRefProvider.Init(ActorSystemImpl system)
   at Akka.Actor.Internal.ActorSystemImpl.Start()
   at Akka.Actor.ActorSystem.CreateAndStartSystem(String name, Config withFallback, ActorSystemSetup setup)
```

### PR DESCRIPTION
**Pull request by yildizoglu**
Fixes #4879

### CODE REVIEW

File: src/core/Akka/Event/Logging.cs [hunk 1]
```diff
@@ -270,7 +270,7 @@ public static LogLevel LogLevelFor(string logLevel)
         {
             if (!string.IsNullOrEmpty(logLevel))
             {
-                logLevel = logLevel.ToUpper();
+                logLevel = logLevel.ToUpper(new System.Globalization.CultureInfo("en-EN"));
```
Comment (Arkatufus) [inline src/core/Akka/Event/Logging.cs:273]: Can you test `ToUpperInvariant` instead?

**Review by Arkatufus — CHANGES_REQUESTED** [review]

File: src/core/Akka/Event/Logging.cs (hunk 1 above)
Comment (yildizoglu) [reply src/core/Akka/Event/Logging.cs:273]: Yes, I just tested it with `ToUpperInvariant ` and worked fine.

**Review by Aaronontheweb — APPROVED** [review]
LGTM

**Comment by Aaronontheweb** [comment]
@Arkatufus looks like https://github.com/akkadotnet/akka.net/pull/4874 still didn't resolve this issue - need to open another PR for that:

```
Akka.Streams.Tests.IO.FileSinkSpec.SynchronousFileSink_should_allow_writing_from_specific_position_to_the_file
Expected cont to be 
"aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb
ccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc
```

**Review by Arkatufus — APPROVED** [review]

**Comment by Arkatufus** [comment]
Yes, that looks good. Thank you for the contribution :)

**Pull request merged**